pybamm/models/base_model.py

```diff
@@ -49,8 +49,26 @@ def __init__(self):
         self.default_parameter_values = pybamm.ParameterValues(
             "input/parameters/lithium-ion/parameters/LCO.csv"
         )
-        mesh = pybamm.FiniteVolumeMacroMesh(self.default_parameter_values, 2)
-        self.default_discretisation = pybamm.FiniteVolumeDiscretisation(mesh)
+
+        self.default_geometry = pybamm.Geometry1DMacro()
+        self.default_parameter_values.process_geometry(self.default_geometry)
```

Comment:
Agree, I just replaced the previous defaults so that the defaults would work in the same way as before. Might be good to make another small issue for this so that it can be dealt with separately from all the other stuff. I have added similar things to the tests.shared so that's where it would go